### PR TITLE
Added command to generate random hex color

### PR DESCRIPTION
**Pull request by irwinj**
Added command to generate a random hex color, formatted "#abc123"

### CODE REVIEW

File: random.py [hunk 1]
```diff
@@ -199,6 +199,15 @@ def generate_email(self):
     def run(self, view, **kwargs):
         self.insert(view, self.generate_email)
 
+class RandomHexColorCommand(RandomText):
+
+    def generate_hex_color(self):
+        #  Code taken from: http://stackoverflow.com/questions/13998901/generating-a-random-hex-color-in-python
+        return '#%06X' % random.randint(0,256**3-1)
```
Comment (kimpettersen) [inline random.py]: Thanks a lot for the PR!
I'm a bigger fan of: `"#%06x" % random.randint(0,0xFFFFFF)` it is a bit easier to understand. Would you mind changing it?

**Comment by kimpettersen** [comment]
Thanks again! It would be great if you could update the README as well

**Comment by irwinj** [comment]
No problem. I've made the requisite changes. Let me know if this works.

**Comment by kimpettersen** [comment]
@irwinj Great, Thank you!

**Pull request merged**